github/filecoin-project.yml

```
@@ -99,7 +99,6 @@ members:
     - FF-FOIT
     - figureouter
     - filecoin-ci
```

Let's restore filecoin-helper here. We intentionally block removing users from the organization via GitHub as Code because this action is hard to revert (it requires an email accept on the revert). Once we merge it, I'm going to manually remove filecoin-helper from the organization and synchronize this repository.

```suggestion
    - filecoin-ci
    - filecoin-helper
```

---

Sounds good.